### PR TITLE
deploy: add more packages to be built for downstream

### PR DESCRIPTION
**Pull request by matz-e**
This should speed up some of our internal CI build times.

### CODE REVIEW

**Comment by matz-e** [comment]
Retest this please

File: deploy/packages/external-libraries.yaml [hunk 1]
```diff
@@ -31,7 +31,6 @@ packages:
       - gmsh+metis~mpi+oce+openmp+shared@4.4.1
       - hdf5~mpi@1.10.5
       - intel-mkl@2019.3.199
-      - jdk@1.8.0_131-b11
```
Comment (pramodk) [inline deploy/packages/external-libraries.yaml]: no longer used?

File: deploy/packages/external-libraries.yaml [hunk 2]
```diff
@@ -47,7 +46,6 @@ packages:
       - qt@5.11.2 ^libjpeg-turbo
       - readline
       - snakemake
-      - spark@2.4.0+hadoop
```
Comment (pramodk) [inline deploy/packages/external-libraries.yaml:50]: why these were moved from externals?

Comment (matz-e) [reply deploy/packages/external-libraries.yaml:50]: Because of environment setting things.

File: deploy/packages/external-libraries.yaml (hunk 1 above)
Comment (matz-e) [reply deploy/packages/external-libraries.yaml]: Upstream dependency. I don't think anyone uses JDK by itself…

Comment (pramodk) [reply deploy/packages/external-libraries.yaml]: We should enable `hpcviewer` and that needs JDK.

Comment (matz-e) [reply deploy/packages/external-libraries.yaml]: Fine.

**Comment by matz-e** [comment]
Retest this please

**Comment by pramodk** [comment]
@matz-e : still relevant??

**Comment by matz-e** [comment]
Totally.

**Comment by matz-e** [comment]
Retest this please

**Review by pramodk — APPROVED** [review]

**Pull request merged**